.github/workflows/update_code.yml

```diff
@@ -25,10 +25,10 @@ jobs:
         run: hub pr checkout ${{ github.event.issue.number }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Set up JDK 11
+      - name: Set up JDK 17
         uses: actions/setup-java@v1
```

Comment:
Why java 17? Aren't we using 11?

Reply:
I upgrade the function runtime to java 17